docs/specification/draft/server/standard-profiles/minimum-authorization-profile.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Minimum Authorization Profile
+---
+
+<div id="enable-section-numbers" />
+
+<Info>**Protocol Revision**: draft</Info>
+
+## Overview
+
+The Minimum Authorization Profile defines the baseline authorization requirements for HTTP-based MCP servers. This profile ensures basic security while maintaining simplicity for initial deployments.
+
+## Profile Identification
+
+This profile is identified by the URL:
+```
+https://modelcontextprotocol.io/specification/draft/server/standard-profiles/minimum-authorization-profile
+```
+
+## MCP Protocol Version
+The minimum MCP protocol version required for any MCP Server to use this profile is "2025-06-18".
+
+## HTTPS Transport
+- MCP servers implementing this profile MUST use HTTPS
+- HTTP connections MUST be rejected or redirected to HTTPS
+- TLS version 1.2 or higher MUST be supported
+
+## OAuth 2.1 support
+- MCP Clients and Servers MUST use OAuth 2.1 for all authorization flows
+
+## Protected Resource Metadata Discovery
```

Comment:
The rest of the section seems to be repeating the authorization specification and security best practices.